tests/scancode/data/unicodepath/unicodepath.expected-linux.json

```diff
@@ -59,20 +57,19 @@
       "type": "file", 
       "name": "%CF%A8%E1%BD%80%E2%8C%A8a", 
       "extension": "", 
-      "date": "2016-12-05", 
```

Comment:
Why's this change necessary in this context? If it was wrong, it should have been wrong before already, or?